src/NextjsBuild.ts

```diff
@@ -250,11 +251,13 @@ export const makeTokenPlaceholder = (value: string): string =>
  * Replaces each path with "../"
  * @param path
  */
-function getRootTracePath(p: string): string {
+function getRootTracePath(nextjsPath: string, p: string): string {
   p = p.replace(/^\.\/?/, '');
-  return p
+  p = p
     .split('/')
     .filter((v) => v && v !== '.')
     .map(() => '..')
     .join('/');
+
+  return path.join(nextjsPath, p);
```

Comment:
not sure I get this
if you want absolute path use path.absolute

Reply:
Hmm good point, I'll change. 
Before, the util function would return "..".

Reply:
wait, is there `path.absolute`? I see `path.relative` and `path.isAbsolute`.

Reply:
nvm `path.resolve` returns the absolute path